src/test/java/edu/ucla/library/libcal/handlers/ProxyHandlerTest.java

```diff
@@ -0,0 +1,92 @@
+
+package edu.ucla.library.libcal.handlers;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import info.freelibrary.util.HTTP;
+import info.freelibrary.util.Logger;
+import info.freelibrary.util.LoggerFactory;
+
+import edu.ucla.library.libcal.Config;
+import edu.ucla.library.libcal.Constants;
+import edu.ucla.library.libcal.MessageCodes;
+import edu.ucla.library.libcal.verticles.MainVerticle;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import io.vertx.core.DeploymentOptions;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.Vertx;
+import io.vertx.ext.web.client.HttpResponse;
+import io.vertx.ext.web.client.WebClient;
+import io.vertx.ext.web.client.predicate.ResponsePredicate;
+import io.vertx.ext.web.codec.BodyCodec;
+import io.vertx.junit5.VertxExtension;
+import io.vertx.junit5.VertxTestContext;
+
+/**
+ * Tests {@link ProxyHandler#handle}.
+ */
+@ExtendWith(VertxExtension.class)
+@TestInstance(Lifecycle.PER_CLASS)
+public class ProxyHandlerTest {
```

Comment:
This is lower priority IMO, but since this test calls the LibCal API, it would be better as an integration test that has a WebClient making a request to the application container that docker-maven-plugin spins up locally.

Reply:
Yes, I'd say let's make this a "phase 2" thing rather than do it now.

Reply:
Yes, I'd say let's save this for a "phase 2" stage.

Reply:
Created SERV-470 for this task. Perhaps we should make a "LibCal Phase 2" epic to collect the "we'll do later" tickets?

Reply:
Chatting with Josh, he'd like to reserve epic use for bigger projects we need planning for, but I will create a "LibCal-Cleanup" ticket and move the remaining non-essential tickets to subtasks under that and then put that into our maintenance epic. Thanks for creating, SERV-470.